tools/topology/topology1/sof/pipe-demux-eq-iir-playback.m4

```diff
@@ -23,17 +23,17 @@ C_CONTROLBYTES(concat(`DEMUX', PIPELINE_ID), PIPELINE_ID,
 define(DEF_EQIIR_COEF, concat(`eqiir_coef_', PIPELINE_ID))
 define(DEF_EQIIR_PRIV, concat(`eqiir_priv_', PIPELINE_ID))
 
-# Bandsplit, created with example_iir_bandsplit.m 25-Aug-2021
+# Bandsplit, created with example_iir_bandsplit.m 26-Oct-2021
 CONTROLBYTES_PRIV(DEF_EQIIR_PRIV,
 `       bytes "0x53,0x4f,0x46,0x00,0x00,0x00,0x00,0x00,'
-`       0xb0,0x00,0x00,0x00,0x00,0x30,0x01,0x03,'
+`       0xb0,0x00,0x00,0x00,0x01,0x20,0x01,0x03,'
 `       0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,'
 `       0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,'
 `       0xb0,0x00,0x00,0x00,0x04,0x00,0x00,0x00,'
 `       0x02,0x00,0x00,0x00,0x00,0x00,0x00,0x00,'
 `       0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,'
 `       0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,'
-`       0x01,0x00,0x00,0x00,0x00,0x00,0x00,0x00,'
+`       0x00,0x00,0x00,0x00,0x01,0x00,0x00,0x00,'
```

Comment:
and this seems the same as the routing, this would need to be product specific?

Reply:
here is a sample so ODM could test the correctness of their hw design before tuning. they need to make their own coefficient and apply it from user space instead of using the default eq setting

Reply:
yes but again this is generic code that isn't supposed to be ODM specific.

Reply:
I thine we don't need to make this odm specific. This coefficient is generated by example_iir_bandsplit.m with default band-splitting function. ODM needs to generate their own coefficient and apply it from user space instead of overwriting this file.

Reply:
It will likely be ODM specific. The band-frequencies are speakers choises dependent and there's also opportunity to add equalization. I've used PIPELINE_FILTERx macro to pass include file names from upper level to pipeline macros.